star/star.c

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Can you write the motive of this program, rather create a readme as well for this directory?